src/Runner.js

```diff
@@ -99,7 +99,7 @@ function getAllFiles(paths, filter) {
     paths.map(file => new Promise((resolve, reject) => {
       fs.lstat(file, (err, stat) => {
         if (err) {
-          process.stdout.error('Skipping path ' + file + ' which does not exist. \n');
```

Comment:
This was giving me errors when testing, as far as i can tell node doesn't have a `stdout.error`. So i just updated to `stderr.write`.